### PR TITLE
Provide Screenshots for the Help Page

### LINKED ISSUE #174: Provide Screenshots
The current Help tab lacks visual guidance, making it difficult for users to fully understand how to navigate and utilize the software. Without step-by-step visual examples, users may struggle with certain functionalities, leading to frustration or missteps. Additionally, all sections are displayed at once, which may overwhelm users, reducing the overall effectiveness of the Help section.

What Must Be Changed:
To improve the usability of the Help tab, visual guidance must be added in the form of screenshots for each section. These screenshots will serve as a step-by-step guide, visually demonstrating how to use the different features of the software. Additionally, the Help tab will implement collapsible sections, where the content is hidden behind clickable buttons, revealing only the relevant information when needed. This will streamline the Help page and provide a more user-friendly, organized experience.

Expected Outcome:
After implementing these changes, the Help tab will display a more interactive and comprehensive visual guide for users. The collapsible content will ensure that users only see the specific instructions they need, while the screenshots will provide clear and intuitive steps to enhance their understanding of how to operate the software.

Acceptance Criteria:

- [ ] Provide clear, high-quality screenshots that demonstrate the steps required to use the File section within the software. Screenshots should be easy to follow, highlighting key features and actions.
- [ ] Include detailed screenshots showing the various steps involved in using the Analysis section. Screenshots should be easy to follow, highlighting key features and actions.
- [ ] Provide screenshots for the Results section, clearly illustrating how to interpret and manage the output of the software.
- [ ] The Help page must have collapsible sections for each part. Each section should be hidden by default, with a button that, when clicked, reveals the content for that section.
- [ ] Only one section’s content should be displayed at a time, hiding all other sections' content automatically.
Additional Notes:
Ensure the buttons are clearly labeled for easy navigation.
The layout should be responsive and user-friendly, adapting to different screen sizes.

### PR DESCRIPTION
**Pull request by BollepalliPranitha**
Fixes #174 

**What was changed?**

The Help tab of the application was updated to include visual guidance in the form of screenshots and an enhanced collapsible section layout.
Specifically, the File, Analysis, and Results sections were updated to include step-by-step visual examples (screenshots) to aid users in understanding how to navigate and use these features.
Collapsible functionality was added so that only the necessary section is shown when a user clicks a corresponding button, reducing clutter and providing a streamlined user experience.
**Why was it changed?**

The existing Help tab lacked visual support and showed all the information at once, overwhelming users and making navigation difficult.
The new design aims to improve usability by offering a more organized experience where users can see only what they need, supplemented with visual examples to clarify key actions and functionalities. This will prevent frustration and make the software easier to use.

**How was it changed?**

Help tab layout: Modified the Help tab to implement collapsible sections using HTML and JavaScript. Now, users click buttons to reveal the content for the File, Analysis, and Results sections, and only one section can be expanded at a time.
Screenshots: Added high-quality screenshots for the following sections:
File section: Shows step-by-step instructions on how to upload and manage files, with screenshots highlighting key actions.
Analysis section: Provides visual guidance on using the analysis features, explaining each step with detailed visuals.
Results section: Demonstrates how to interpret the output with examples of charts, tables, and key information to look for.

**How was it tested**

Manual testing was performed to ensure that the collapsible sections open and close as expected, and only one section is visible at a time.
Tested across multiple screen sizes to verify responsiveness.
Screenshots were reviewed for clarity, and the overall Help tab layout was tested for usability.

**Screenshots that show the changes (if applicable):**

image
image
image
image
image

### CODE REVIEW

File: code/ui.R [hunk 1]
```diff
@@ -282,70 +292,109 @@ ui <- navbarPage(
...
-              The tables can be thought of as layers, with chunks for each sample layered on top of oneanother.
-              For each table, you may choose how many entries to show and navigate through pages. At the very top of the screen, there is a navigation bar, where there are four options- File, Analysis, Results, and Help.
-              The Analysis and Results tabs will remain inactive until you have indicated you are done configuring inputs and uploading datasets. The help page will always be available should you need to reference it at any point. The upload tab under the File menu will also be available after you have indicated you are done uploading. The inputs on the left will be disabled, but you can still
-              view and interact with the datasets for reference."),       
+        HTML("The program will open to the Upload page. This page is divided in two, a panel on the left for inputs and a panel on the right for the processed datasets."),
+        div(img(src = "Upload_Processed.png", class="img-half", alt = "Screenshot showing the general information of the upload page", style = "clear:both")),
```
Comment (adhampton110) [inline code/ui.R:307]: I see you are using inline-styling for your images, specifically for your `img` tags. You may be able to get away by using a CSS rule to define the necessary styling needed for the `img` elements. 

For example: 
```css
img {
 /* styling goes here */
}
```

**Review by adhampton110 — COMMENTED** [review]
Left a couple of comments...overall nice job! It may be beneficial to showcase the modifications of the help page to the clients to get better feedback.

File: code/ui.R (hunk 1 above)
Comment (BollepalliPranitha) [reply code/ui.R]: I have improved the styling by shifting inline styles to internal styles, applying consistent styling to all img{} elements for better maintainability. However, certain images require unique styles, which I’ve addressed using class selectors and inline styling to ensure they retain their specific appearance. These targeted styles have been given higher priority, ensuring that future changes to the general styling will not affect them. This approach maintains flexibility while providing a clear separation between common and custom image styles.

File: code/ui.R [hunk 2]
```diff
@@ -1,12 +1,22 @@
 # ui.R is essentially the skeleton of the application and its visual component. 
 # It is populated and dynamically changed by server.R
-
+library(shinyjs)
```
[the remark this answers is not in the document]
Comment (BollepalliPranitha) [reply code/ui.R]: It worked -- shifted this line to app.R

**Review by adhampton110 — APPROVED** [review]
Looks good to me

**Pull request merged**